addons/stock/migrations/8.0.1.1/post-migration.py

```diff
@@ -483,6 +483,47 @@ def get_location_by_usage(usage):
             'sequence': max_sequence + 2,
         })
 
+    # If warehouse is main warehouse, we have to create records in
+    # ir_model_data, because these are used in testing:
+    def create_missing(cr, xml_id, res_id):
+        """Add missing record to ir_model_data."""
+        model_data_obj = registry['ir.model.data']
+        expected_record = model_data_obj.search(
+            cr, uid, [('module', '=', 'stock'), ('name', '=', xml_id)]
+        )
+        if not expected_record:
+            model_data_obj.create(
+                cr, uid, {
+                    'module': 'stock',
+                    'model': 'stock.picking.type',
+                    'name': xml_id,
+                    'res_id': res_id,
+                }
+            )
+        else:
+            if res_id != expected_record[0].res_id:
+                # autocorrect existing ir_model_data:
+                model_data_obj.write(
+                    cr, uid, [expected_record[0].id], {'res_id': res_id}
+                )
+                logger.warn(
+                    "xml_id %s now points to res_id %d, no longer to %d.",
+                    xml_id, res_id, existing_id
+                )
+
+    model_data_obj = registry['ir.model.data']
+    main_warehouse = model_data_obj.search(
+        cr, uid, [('module', '=', 'stock'), ('name', '=', 'warehouse0')]
+    )
+    if not main_warehouse:
```

Comment:
it's an error condition if this doesn't exist, as the record must be created during the update: https://github.com/OCA/OCB/blob/8.0/addons/stock/stock_data.xml#L147 (not not noupdate). So I think we can leave this check out